manageiq-providers-ibm_power_hmc.gemspec

```diff
@@ -19,7 +19,7 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
 
-  spec.add_dependency "ibm_power_hmc", "~> 0.12.0"
+  spec.add_dependency "ibm_power_hmc", "~> 0.14.0"
```

Comment:
If I cherry-pick https://github.com/IBM/ibm_power_hmc_sdk_ruby/pull/82/commits/3fc0b69ca96aaa6d1dff998cd1bceed7c6c67cab directly onto `v0.12.0` of the `ibm_power_hmc` gem, then specs pass. Applying the patch to the _main_ branch would require us to jump from `v0.12.0` -> `v0.14.0` and requires that the refresh spec VCR be recorded again.

Reply:
Perhaps the best path would be to release it as `v0.12.1`?

Reply:
Usually a backwards-compatible interface addition would be a minor version bump (https://semver.org/), but this could easily be seen as a bug fix as well so no issues with `v0.12.1` especially if we want to backport this.